docs/RouteRequest.md

```diff
@@ -15,9 +15,9 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 
 | Config Parameter                                                                                             |          Type          | Summary                                                                                                                                        |  Req./Opt. | Default Value    | Since |
 |--------------------------------------------------------------------------------------------------------------|:----------------------:|------------------------------------------------------------------------------------------------------------------------------------------------|:----------:|------------------|:-----:|
-| [alightSlack](#rd_alightSlack)                                                                               |       `duration`       | The minimum extra time after exiting a public transport vehicle.                                                                               | *Optional* | `"PT0S"`         |  2.0  |
+| [alightSlack](#rd_alightSlack)                                                                               |       `duration`       | The extra time after exiting a public transport vehicle.                                                                                       | *Optional* | `"PT0S"`         |  2.0  |
```

Comment:
I realized I've used the word minimum in https://github.com/opentripplanner/OpenTripPlanner/pull/5185 as well. I don't know what was the original goal with using that term.